packages/dashboard/src/lib/components/shared/configurable-operation-arg-input.tsx

```
@@ -10,42 +19,374 @@ export interface ConfigurableOperationArgInputProps {
-    if ((definition.ui as any)?.component === 'facet-value-form-input') {
-        return <FacetValueInput value={value} onChange={onChange} readOnly={readOnly} />;
+    const uiComponent = (definition.ui as any)?.component;
+    const argType = definition.type as ConfigArgType;
+    const isList = definition.list ?? false;
+
+    // Handle specific UI components first
+    if (uiComponent) {
+        switch (uiComponent) {
+            case 'product-selector-form-input':
+                const entityType =
+                    (definition.ui as any)?.selectionMode === 'variant' ? 'ProductVariant' : 'Product';
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)